src/vm/eventtracepriv.h

```diff
@@ -412,9 +412,13 @@ class BulkStaticsLogger
 class XplatEventSourceLogger
 {
 public:
+#ifdef FEATURE_EVENT_TRACE
```

Comment:
This conditional compilation is done for FireEtw* and XplatEventLogger::IsEventLoggingEnabled under the hoods. If it is not pressing to check in the change as it is please do look to build on top of https://github.com/ramarag/coreclr/commit/84a8d4d375dea0c59eabbae11e3e68d9532e23f1